### ISSUE TITLE
Add more sounds?

### ISSUE THREAD

**Issue by matsn0w** [Walsheyy/WMServerSirens#8]
We're now having 8 sound options (`alpha`, `bravo`, `charlie`, `delta`, `echo`, `foxtrot`, `golf` and `hotel`), is it possible to add more? Like `india`, `juliet`, `kilo`, `lima`...

I tried it myself but I'm having trouble figuring out how the hashes work etc.

**Comment by mrquentinh** [comment]
@matsn0w
> We're now having 8 sound options (`alpha`, `bravo`, `charlie`, `delta`, `echo`, `foxtrot`, `golf` and `hotel`), is it possible to add more? Like `india`, `juliet`, `kilo`, `lima`...
> 
> I tried it myself but I'm having trouble figuring out how the hashes work etc.

It is possible. Try to found the page about custom engine sound on gta5 mods forum and you will understand more. If not i try to make my audio pack for my server today. I can tell you if it is hard or not, help you, etc..

**Comment by matsn0w** [comment]
Nice! Do you have a link for that?

**Comment by mrquentinh** [comment]
> Nice! Do you have a link for that?

Yup, [here](https://forums.gta5-mods.com/topic/31272/tutorial-how-to-create-an-add-on-engine-sounds-dlc) it can help you like me to make your own audio pack. I have made my own like i said before and it work after read that page (on the here text is a link to) and manage the file actualy in resource to make all perfect

**Comment by matsn0w** [comment]
Thanks! I'm going to have a dive into that article.

**Comment by Walsheyy** [comment]
I'm going to close this issue now since it has been resolved.

**Comment by DevKilo1** [comment]
I could use some help with this if anybody wouldn't mind helping. I updated the nametable and the rel dat file, I edited the oac file to incorporate the new sound files, that works fine. I believe I have done everything that is needed, but something's not working. The sounds aren't being loaded. It's not the ELS script I'm using, I have the alpha-hotel tones working. I'm trying to add more than just 8 tones. Here's my resource, let me know what I'm missing here. @matsn0w @Walsheyy 
[wm-serversirens.zip](https://github.com/Walsheyy/WMServerSirens/files/9998274/wm-serversirens.zip)

**Comment by boghilife** [comment]
@DevKilo1 Did you ever find out what the problem was?

**Comment by DevKilo1** [comment]
> @DevKilo1 Did you ever find out what the problem was?

Yes, I did. It had to do with my dat.rel file. Some areas of it need to be hex encrypted and the remaining can be in plain text. It's very technical and complicated. I ended up rewriting the whole file by hand so that I could actually know what entries refer to what file. Also note that once FiveM loads the file it will become completely encrypted automatically. I'd be happy to provide my modified copy of the dat.rel.xml file if that would help you understand how yours must be.
